package.json

```diff
@@ -38,11 +38,9 @@
     "@types/react": "^18",
     "@types/react-dom": "^18",
     "@types/sprintf-js": "^1.1.4",
-    "@types/wordpress__api-fetch": "^3.23.1",
-    "@types/wordpress__blob": "^2.8.0",
     "@types/wordpress__block-editor": "^11.5.15",
     "@types/wordpress__blocks": "^12.5.13",
-    "@types/wordpress__dom-ready": "^2.9.0",
+    "@wordpress/api-fetch": "^7.4.0",
```

Comment:
@kuuak But this means that we import `@wordpress` package inside our final JS on the frontend no ? I don't know if we can find an alternative to this ?

Reply:
@charl0tee It's a devDependency therfore it should not be included in the final JS. I will validate and check to be 100% sure about that

Reply:
@charl0tee I can validate that it doesn't leak to the client JS

|   | before packages changes | after |
| --- | --- | --- |
| Nextjs  | ![Capture d’écran 2024-07-29 à 16 43 29](https://github.com/user-attachments/assets/8a29791c-4b43-44eb-ba33-310b7e98119c) | ![Capture d’écran 2024-07-29 à 16 51 46](https://github.com/user-attachments/assets/0c6f68a6-5c0a-492c-b0d9-0c5120ac28ef) |
| WordPress editor | ![Capture d’écran 2024-07-29 à 16 46 55](https://github.com/user-attachments/assets/f89858e1-2103-46c4-9e0b-2e28f3dc8226) | ![Capture d’écran 2024-07-29 à 16 52 58](https://github.com/user-attachments/assets/2d88d17c-f21b-4e21-9659-a0b4b71a73d9) |

Reply:
@kuuak Aaah, nice ! 👌🏽